### DISCUSSION TITLE
[Help]: Support mouse/trackpad horizontal slide

### DISCUSSION THREAD

**Discussion by rafaell-lycan** [davidjerleke/embla-carousel#870] [Help] [answered]
### Summary

Hey there 👋 

I'm trying `embla-carousel` and the only deal breaker I'm facing now is how to support the trackpad horizontal scroll event. [example](https://codesandbox.io/p/sandbox/element-horizontal-scroll-progress-4cd4v4?file=%2Fsrc%2Findex.js&from-embed=)

Am I missing something?

For context, I'm already using `dragFree: true` for mobile, and it works like a charm 👌 

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by sarussss** [comment, marked as answer]
Hi @rafaell-lycan [is this what you need](https://www.embla-carousel.com/plugins/wheel-gestures/)
